app/visits/reports/edit/template.hbs

```diff
@@ -181,13 +181,12 @@
             <span class="test-visit-discharge-date-data">{{date-format model.reportDate format="DD/MM/YYYY"}}</span>
```

Comment:
I didn't notice this before -- this shouldn't be specifying a date format, it should use the default for the data-helper, which is the locale specific date.  Please take off the ```format="DD/MM/YYYY"```